dist/js/medium-editor.js

```diff
@@ -2468,13 +2468,24 @@ var AnchorPreview;
             this.instance_handleAnchorMouseout = null;
         },
 
-        handleEditableMouseover: function (event) {
-            if (event.target && event.target.tagName.toLowerCase() === 'a') {
+       handleEditableMouseover: function (event) {
+            var target;
+
+             if(event.target){
+
+                if(event.target.tagName.toLowerCase() === 'a'){
+                    var target = event.target;
+                }else if(event.target.parentNode.tagName.toLowerCase() === 'a'){
```

Comment:
What if the text is bold **and** italic ?
Maybe you can take some inspiration from the [`setTargetBlank`](https://github.com/daviferreira/medium-editor/blob/master/src/js/util.js#L322-L333).